### ISSUE TITLE
[FEATURE]: System Authentication / Fingerprint Auth (without storing)

### ISSUE THREAD

**Issue by LogGits** [AndresMorelos/InvoncifyReleases#88]
I'd like to be able to use system auth to login (e.g. system password or fingerprint biometric auth). It would need to work such that it doesn't store the data in the app. I saw [this](https://www.electronjs.org/docs/latest/api/system-preferences#systempreferencesprompttouchidreason-macos) for mac but never got a chance to test out/implement.

Loving the app, keep up the great work 🙂

**Comment by AndresMorelos** [comment]
@LogGits Do you mean auth to the app using Biometric or System auth, and store the data (Invoices, contacts, etc) in a server?

**Comment by LogGits** [comment]
> @LogGits Do you mean auth to the app using Biometric or System auth, and store the data (Invoices, contacts, etc) in a server?

No, that could be cool in the future but I'm referring to the ability to use the mac's touchid or if that doesnt exist the system password to login to the app (e.g. the same thing that happens when you want to unlock the computer) instead of the Invoncify login form.
